tests/sphinx/test_parsers.py

```diff
@@ -44,7 +44,7 @@ def test_waves(self):
         waves_path = Path.joinpath(test_path,'static/sphinx/sphinx_test_waves')
         waves = SphinxWavesReader('waves.sxb',cwd=waves_path)
         self.assertTrue(waves.nk==1)
-        self.assertTrue(np.mean(waves.get_psi_rec(0,0,0))>0)
+        self.assertTrue(np.mean(np.abs(waves.get_psi_rec(0,0,0)))>0)
```

Comment:
That test could only fail if all values in the waves are exactly zero?

Reply:
It is unlikely for the waves file, it is for a bulk Al supercell, which I created.

Reply:
And for the Al case, it is ok if the value of `waves.get_psi_rec(0,0,0)` are negative? I just think the test now is really weak and I am not sure if this is intended. I mean a single non-zero value in the full `waves.get_psi_rec(0,0,0)` would immediately cause a positive mean overall.

Reply:
But my understanding was to make sure the reading routine was correctly reading the waves file.

Reply:
Yes, the test would also fail if there is an error, however, the shape or some more specific value would test the _correct_ reading better than 'have one not zero value in that array'

Reply:
I violently committed my suggestion 😎